app/routes/events/view/export.js

```diff
@@ -1,9 +1,20 @@
 import Ember from 'ember';
 
-const { Route } = Ember;
+const { Route, RSVP, Object } = Ember;
 
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Export');
+  },
+  model() {
+    return RSVP.hash({
+      exportData: Object.create({
```

Comment:
This model is not needed at all. There is no data retrieval from server and undefined variables are _falsey_ by default in ember.